proto/babylon/incentive/tx.proto

```diff
@@ -26,11 +27,13 @@ service Msg {
 // MsgWithdrawReward defines a message for withdrawing reward of a stakeholder.
 message MsgWithdrawReward {
   option (cosmos.msg.v1.signer) = "address";
-  // {finality_provider, btc_delegation}
-  string type = 1;
+  // RESERVED: previously stakeholder type was a string. Now is an enum
+  reserved 1;
   // address is the address of the stakeholder in bech32 string
   // signer of this msg has to be this address
   string address = 2;
+  // Type of stakeholder (FINALITY_PROVIDER or BTC_DELEGATION)
+  StakeholderType type = 3;
```

Comment:
According to #581, we also want to change `BTC_DELEGATION` to `BTC_STAKER`?

Reply:
done